app/lib/actions/candlepin/product/content_add.rb

```diff
@@ -10,7 +12,17 @@ class ContentAdd < Candlepin::Abstract
 
         def run
           output[:response] = ::Katello::Resources::Candlepin::Product.
-              add_content(input[:owner], input[:product_id], input[:content_id], true)
+              add_content(input[:owner], input[:product_id], input[:content_id], ENABLED)
```

Comment:
This is what sets it enabled in Candlepin (`enabled: 0` or `enabled: 1` will show in subscription-manager repos)